src/stories/Buttons.stories.tsx

```diff
@@ -42,9 +42,9 @@ Tertiary.args = {
   color: 'tertiary',
   size: 'small',
 }
-Primary.argTypes = {
+Tertiary.argTypes = {
   size: {
     control: 'select',
-    options: ['small', 'medium', 'large'] as unknown as ButtonTypeMap['props']['size'],
```

Comment:
I assume it was meant to be 
`options: ['small', 'medium', 'large'] as ButtonTypeMap['props']['size'][],`
to get code-completion inside the array.

`satisfies` gives us code-completion, but also prevents typos.

Reply:
Awesome, wasn't aware of `satisfies`.